lib/api.js

```diff
@@ -1,6 +1,6 @@
 module.exports = adminsApi
 
-var clone = require('lodash.clone')
+var clone = require('lodash')
```

Comment:
this should be `lodash/clone` instead of `lodash`. The tests should have failed, I’ll look into it